projects/smart-table/src/lib/components/smart-table/smart-table.component.ts

```diff
@@ -405,9 +407,15 @@ export class SmartTableComponent implements OnInit, OnDestroy {
 
   public exportToExcel() {
     this.pageChanging = true;
-    this.dataQuery$.pipe(
-      first(),
-      switchMap(dataQuery => this.dataService.getAllData(this.apiUrl, this.httpHeaders, dataQuery)),
+    let dataQuery = this.activeDataQuery;
+    if (!this.moduleConfig.options || !this.moduleConfig.options.exportWithFilters) {
+      dataQuery = {
```

Comment:
Wat blijft er nog over van de "...this.activeDataQuery" als je de filters en de sort overschrijft?